debian/enterprise-control-center/Dockerfile

```diff
@@ -33,7 +33,7 @@ ENV CONTROL_CENTER_CONFIG_DIR=/etc/confluent-${COMPONENT}
 EXPOSE 9021
 
 RUN echo "===> Installing ${COMPONENT}..." \
-    && apt-get update && apt-get install -y confluent-${COMPONENT}=${CONFLUENT_VERSION}-${CONFLUENT_DEB_VERSION} \
+    && apt-get update && apt-get install -y confluent-${COMPONENT}=${CONFLUENT_VERSION}${CONFLUENT_PLATFORM_LABEL}-${CONFLUENT_DEB_VERSION} \
```

Comment:
Ok, this was the explanation I was looking for. I guess it's unlikely anybody else does snapshot builds, but just sticking that as a comment above the variables in the `Makefile` would really clarify what they are for.